docs/security.html

```diff
@@ -36,7 +36,104 @@ <h3 class="anchor-heading"><a id="security_overview" class="anchor-link"></a><a
+      In the following sections, this document covers how to configure the remaining protocols.</p>
+    
+    <p>If each required listener uses a separate security protocol, it is also possible to use the
+      security protocol name as the listener name in <code>listeners</code>. Using the example above,
+      we could skip the definition of the <code>CLIENT</code> and <code>BROKER</code> listeners
+      using the following definition:</p>
+    
+    <pre class="line-numbers"><code class="language-text">listeners=SSL://localhost:9092,PLAINTEXT://localhost:9093</code></pre>
+      
+    <p>However, we recommend users to provide explicit names for the listeners since it
+      makes the intended usage clearer.</p>
+
+    <p>Among the listeners in this list, it is possible to declare the listener to be used for
+      inter-broker communication by setting the <code>inter.broker.listener.name</code> configuration
+      to the name of the listener. If not defined, then the inter-broker listener is determined
+      by the security protocol defined by <code>security.inter.broker.protocol</code>, which
+      defaults to <code>PLAINTEXT</code>.</p>
+    
+    <p>For legacy clusters which rely on Zookeeper to store cluster metadata, it is possible to
+      declare a separate listener to be used for metadata propagation from the active controller
+      to the brokers. This is defined by <code>control.plane.listener.name</code>. The active controller
+      will use this listener when it needs to push metadata updates to the brokers in the cluster.
+      The benefit of using a control plane listener is that it uses a separate processing thread,
+      which makes it less likely for application traffic to impede timely propagation of metadata changes
+      (such as partition leader and ISR updates). Note that the default value is null, which
+      means that the controller will use the same listener as defined by <code>inter.broker.listener</code></p>
+    
+    <p>In a KRaft cluster, the listener defined by <code>inter.broker.listener.name</code> is used
+      exclusively for requests sent to nodes which have the "broker" role enabled in <code>process.roles</code>.
+      For legacy clusters, the active controller will connect to all brokers in the cluster
+      to send metadata updates. In KRaft, it is the other way around. The broker is responsible
+      for finding the current controller and connecting to it in order to receive metadata updates.
+      Hence the configuration <code>control.plane.listener.name</code> cannot be used in KRaft clusters.</p>
+      
+    <p>Instead, nodes with the "controller" role must define separate listeners through the
+      <code>controller.listener.names</code> configuration. Any node running with both the
+      "broker" and "controller" roles must use separate listeners for the broker and controller.
+      For example, a minimal configuration for a combined node might look like the following:</p>
```

Comment:
```suggestion
    <p>Instead, servers with the <code>controlle</code> role must define separate listeners through the
      <code>controller.listener.names</code> configuration. Any server running with both the
      <code>broker</code> and <code>controller</code> roles must use separate listeners for the broker and controller.
      For example, a minimal configuration for a combined server might look like the following:</p>
```

The broker also needs to the controller listener name so that it knows how which security protocol to use.